react/Empty/index.jsx

```diff
@@ -4,17 +4,36 @@ import PropTypes from 'prop-types'
 import Icon, { iconPropType } from '../Icon'
 import styles from './styles.styl'
 import Typography from '../Typography'
+import createDepreciationLogger from '../helpers/createDepreciationLogger'
+
+const depreciationLogger = createDepreciationLogger()
 
 export const Empty = ({
   icon,
   title,
   text,
   children,
   className,
+  layout,
   ...restProps
 }) => {
+  if (layout) {
+    depreciationLogger(
+      'The use of c-empty style is deprecated, Empty component must not have a defined layout'
```

Comment:
I think it'd be more explicit with "Please add layout=false in the props of the Empty component, and implement the vertical centering with a custom layout component".

Reply:
You're right